packages/mdc-snackbar/README.md

```diff
@@ -12,17 +12,17 @@ path: /catalog/snackbars/
 <!--<div class="article__asset">
   <a class="article__asset-link"
      href="https://material-components.github.io/material-components-web-catalog/#/component/snackbar">
-    <img src="{{ site.rootpath }}/images/mdc_web_screenshots/snackbars.png" width="336" alt="Snackbars screenshot">
+    <img src="{{ site.rootpath }}/images/mdc_web_screenshots/snackbars.png" width="336" alt="Snackbar screenshot">
   </a>
 </div>-->
 
 Snackbars provide brief messages about app processes at the bottom of the screen.
 
-## Design & API Documentation
+## Design & Demo
```

Comment:
This is a change we should apply across all readmes in this format if we do this. I'd leave this out of this PR.

Reply:
Done